components/bodytext/BodyText.js

```diff
@@ -5,9 +5,11 @@ const BodyText = styled.p`
   font-size: var(--body-text);
   font-weight: var(--regular);
   width: 85%;
+  // FIXME: Remove important;
   margin-top: 20px !important;
 
-  ${media.between("medium", "large")`
```

Comment:
The way you've implemented your responsive design is counterintuitive. Instead of starting with the smallest screen as the default (Mobile first), you're working with desktop then applying media queries for mobile and for super large screens. Writing media queries like these are only going to cause you more issues than it solves.